src/Commands/TerminusCommand.php

```diff
@@ -21,21 +24,34 @@
      * @var boolean True if the command requires the user to be logged in
      */
     protected $authorized = false;
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
 
     /**
      * TerminusCommand constructor
-     *
-     * @param Config $config Terminus configuration object
      */
-    public function __construct($config)
+    public function __construct()
     {
-        $this->config = $config;
+        $this->config = new Config();
+        $this->setLog(new ConsoleLogger($this->output()));
```

Comment:
I should also mention that Robo sets up its own logger object, so if you just make TerminusCommand implement LoggerAwareInterface, then Robo will inject a working logger for you, available via `protected $logger;`, which is in LoggerAwareTrait.